installing/installing_aws/installing-aws-customizations.adoc

```diff
@@ -17,30 +17,16 @@ The scope of the {product-title} installation configurations is intentionally na
 
 == Prerequisites
 
-* Review details about the
-xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update]
-processes.
-* xref:../../installing/installing_aws/installing-aws-account.adoc#installing-aws-account[Configure an AWS account]
-to host the cluster.
+* You reviewed details about the xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update] processes.
+* You read the documentation on xref:../../installing/installing-preparing.adoc#installing-preparing[selecting a cluster installation method and preparing it for users].
+* You xref:../../installing/installing_aws/installing-aws-account.adoc#installing-aws-account[configured an AWS account] to host the cluster.
 +
 [IMPORTANT]
 ====
-If you have an AWS profile stored on your computer, it must not use a temporary
-session token that you generated while using a multi-factor authentication
-device. The cluster continues to use your current AWS credentials to create
-AWS resources for the entire life of the cluster, so you must use long-lived
-credentials. To generate appropriate keys, see
-link:https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html[Managing Access Keys for IAM Users]
-in the AWS documentation. You can supply the keys when you run the installation
-program.
+If you have an AWS profile stored on your computer, it must not use a temporary session token that you generated while using a multi-factor authentication device. The cluster continues to use your current AWS credentials to create AWS resources for the entire life of the cluster, so you must use long-lived credentials. To generate appropriate keys, see link:https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html[Managing Access Keys for IAM Users] in the AWS documentation. You can supply the keys when you run the installation program.
```

Comment:
I am assuming that all of these changes to the Important admonitions you just removed the line breaks.

Reply:
@mburke5678 thank you for your review. Yes, that is correct. I am removing the text wrapping while I'm in this section.